spec/support/api_responses/foursquare/recent_checkins/checkins.json

```diff
@@ -0,0 +1 @@
+{"meta":{"code":200,"requestId":"58b8a2544c1f6737c63919a2"},"notifications":[{"type":"notificationTray","item":{"unreadCount":3}}],"response":{"recent":[{"id":"58b8a13b2f91cb643287800e","createdAt":1488494907,"type":"checkin","timeZoneOffset":-180,"user":{"id":"11776920","firstName":"Pardal","gender":"male","relationship":"friend","photo":{"prefix":"https:\/\/igx.4sqi.net\/img\/user\/","suffix":"\/4C1Y5C5KUALZP533.jpg"}},"venue":{"id":"571f9fa5498ed7fefba8e613","name":"JoJo Ramen","contact":{"phone":"+551132795005","formattedPhone":"+55 11 3279-5005"},"location":{"address":"R. Dr. Rafael de Barros, 262","lat":-23.572845801116415,"lng":-46.64705497838687,"labeledLatLngs":[{"label":"display","lat":-23.572845801116415,"lng":-46.64705497838687}],"postalCode":"04003-041","cc":"BR","city":"São Paulo","state":"SP","country":"Brazil","formattedAddress":["R. Dr. Rafael de Barros, 262","São Paulo, SP","04003-041"]},"categories":[{"id":"55a59bace4b013909087cb24","name":"Ramen Restaurant","pluralName":"Ramen Restaurants","shortName":"Ramen","icon":{"prefix":"https:\/\/ss3.4sqi.net\/img\/categories_v2\/food\/ramen_","suffix":".png"},"primary":true}],"verified":false,"stats":{"checkinsCount":1821,"usersCount":1034,"tipCount":134},"url":"http:\/\/www.jojoramen.com.br","allowMenuUrlEdit":true,"beenHere":{"lastCheckinExpiredAt":0},"specials":{"count":0}},"source":{"name":"Swarm for iOS","url":"https:\/\/www.swarmapp.com"},"photos":{"count":0,"items":[]},"posts":{"count":0,"textCount":0},"likes":{"count":0,"groups":[]},"like":false,"comments":{"count":0},"sticker":{"id":"57fbaacc498e4805f8ea2971","name":"Noodles","image":{"prefix":"https:\/\/irs3.4sqi.net\/img\/sticker\/","sizes":[60,94,150,300],"name":"\/ramen_bb433a.png"},"stickerType":"unlockable","group":{"name":"collectible","index":72},"pickerPosition":{"page":3,"index":0},"teaseText":"Slurp!","unlockText":"Udon need to hear it from us. Ramen is pho-nomenal\u2026.we\u2019ll let you stew on that one. #souppuns","bonusText":"Use at Ramen Restaurant, Soba Restaurants, Udon Restaurants, Noodle Houses for a bonus.","points":2,"bonusStatus":"Use once per week. Recharges Sunday at midnight."},"isMayor":false}]}}
```

Comment:
Você pode deixar esse arquivo formatado, assim fica fácil para quem for ler, inclusive o atom formata se quiser, tem um comando `Json Prettify`, algo assim.